src/features/bookings/api/index.ts

```diff
@@ -1 +1,5 @@
 export { useUserHasBookings } from './useUserHasBookings'
+export { useArchiveBookingMutation } from './useArchiveBookingMutation'
+export { useBookings } from './useBookings'
+export { useOngoingOrEndedBooking } from './useOngoingOrEndedBooking'
+export { useEndedBookingFromOfferId } from './useEndedBookingFromOfferId'
```

Comment:
Sur le (standard)[https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/codeStructure.md], on a pas noté d'`index`. C'est potentiellement à changer mais faut que ce soit spécifier si c'est le cas je pense :)

Reply:
Yes tout à fait, comme on l'a dit ce matin au point ts-prune je suis partante pour mettre des index qui exportent explicitement des fonctions ou composants, je l'ajoute au standard